StarkInfra/StarkInfra/CreditNote/CreditNote.cs

```diff
@@ -17,14 +16,14 @@ namespace StarkInfra
     /// <br/>
     /// Properties:
     /// <list>
-    ///     <item>TemplateID [string]: ID of the contract template on which the credit note will be based. ex: templateId="0123456789101112"</item>
-    ///     <item>Name [string]: credit receiver's full name. ex: name="Edward Stark"</item>
+    ///     <item>TemplateID [string]: ID of the contract template on which the credit note will be based. ex: "0123456789101112"</item>
+    ///     <item>Name [string]: credit receiver's full name. ex: "Edward Stark"</item>
     ///     <item>TaxID [string]: credit receiver's tax ID (CPF or CNPJ). ex: "20.018.183/0001-80"</item>
-    ///     <item>NominalAmount [integer]: amount in cents transferred to the credit receiver, before deductions. ex: nominalAmount=11234 (= R$ 112.34)</item>
-    ///     <item>Scheduled [DateTime or string]: date of payment execution. ex: scheduled=datetime(2020, 3, 10)</item>
+    ///     <item>NominalAmount [integer]: amount in cents transferred to the credit receiver, before deductions. ex: 11234 (= R$ 112.34)</item>
+    ///     <item>Scheduled [DateTime or string]: date of payment execution. ex: datetime(2020, 3, 10)</item>
     ///     <item>Invoices [list of CreditNote.Invoice objects]: list of Invoice objects or dictionaries to be created and sent to the credit receiver. ex: new List<CreditNote.Invoice>{ new CreditNote.InvoiceInvoice() }</item>
-    ///     <item>Payment [CreditNote.Transfer object]: Transfer object or dictionary to be created and sent to the credit receiver. ex: payment=Transfer()</item>
-    ///     <item>Signers [list of Signer objects]: list of Signer objects containing signer's information</item>
+    ///     <item>Payment [CreditNote.Transfer object]: Transfer object or dictionary to be created and sent to the credit receiver. ex: Transfer()</item>
```

Comment:
Docstring do parâmetro **Payment** para: `payment entity to be created and sent to the credit receiver`